Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/app/plugin/assembler/sleigh/symbol/AssemblyNumericTerminal.java

```diff
@@ -63,7 +63,7 @@ public String toString() {
 	/**
 	 * This is only a convenience for testing
 	 * 
-	 * Please use {@link #match(String, AssemblyGrammar, Map<String, Long>)}.
+	 * Please use {@link #match(String, int, AssemblyGrammar, Map) match(String, int, AssemblyGrammar, Map&lt;String, Long&gt;)}.
```

Comment:
html escaping is required within the `{@link}` attempting to use `{@code}` or `{@literal}` will not work.